lib/svg-sprite.js

```diff
@@ -220,6 +220,23 @@ SVGSpriter.prototype.compile = function(config, cb) {
     this._compile();
 };
 
+/**
+ *
+ * @param {object} config                                   Configuration
+ * @returns {Promise<{result:{object}, data:{object}}>}     Results
+ */
+SVGSpriter.prototype.compileAsync = function(config) {
```

Comment:
This is new. Should be added in a new branch, or at least a clear patch, but I'd still prefer a separate PR so that it's clearly stated in the release notes.

Reply:
Ok! I'll do it tomorrow